src/app/config/confparse.h

```diff
@@ -41,8 +41,8 @@ typedef int (*validate_fn_t)(void*,void*,void*,int,char**);
 
 struct config_mgr_t;
 
-/** Callback to free a configuration object. */
-typedef void (*free_cfg_fn_t)(const struct config_mgr_t *mgr, void*);
+/** Callback to clear all non-managed fields of a configuration object. */
```

Comment:
Please explain what a non-managed field is.

This comment should not block this PR, but I need to understand what a non-managed field is.

Reply:
Ok. Do we need unit tests that have non-managed fields?

Reply:
@nmathewson do we need unit tests that have non-managed fields?

Reply:
This looks like beaf7305a0e18304a395faae5dd79bf1f94d1160?